src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/RichTextEditorIT.kt

```diff
@@ -0,0 +1,163 @@
+package com.atlassian.performance.tools.jiraactions.api.scenario
+
+import com.atlassian.performance.tools.dockerinfrastructure.api.browser.DockerisedChrome
+import com.atlassian.performance.tools.dockerinfrastructure.api.jira.JiraCoreFormula
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import com.atlassian.performance.tools.jiraactions.api.ActionResult
+import com.atlassian.performance.tools.jiraactions.api.SeededRandom
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.*
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.measure.output.CollectionActionMetricOutput
+import com.atlassian.performance.tools.jiraactions.api.memories.User
+import com.atlassian.performance.tools.jiraactions.api.memories.UserMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveIssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveIssueMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveJqlMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveProjectMemory
+import com.atlassian.performance.tools.jiraactions.api.w3c.DisabledW3cPerformanceTimeline
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import java.time.Clock
+import java.util.*
+
+/**
+ * Test Editing issue without SetupAction (i.e Rich Text editor enabled)
+ * Covers:
+ *      EditIssue
+ *      AddComment
+ */
+class RichTextEditorIT {
+
+    @Test
+    fun shouldRunScenarioWithoutErrors() {
```

Comment:
Test code is not only about verifying the state of the application. It's also living documentation (IMO the best one because it can't be outdated). Please consider making the test more readable. 
See: http://wiki.c2.com/?ArrangeActAssert

Reply:
I see `runTestOnOneInstance` is hidden in runTestOnOneInstance and the test isn't a test, it runs a suite of tests. Consider ignoring my previous comment.